airbyte-integrations/connectors/source-looker/source_looker/spec.json

```diff
@@ -9,9 +9,15 @@
     "properties": {
       "domain": {
         "type": "string",
-        "pattern": ["^[a-zA-Z0-9._-]*\\.looker\\.com$"],
-        "examples": ["domainname.looker.com"],
-        "description": "Domain for your Looker account, e.g. airbyte.cloud.looker.com"
+        "pattern": [
+          "(^[a-zA-Z0-9._-]*\\.looker\\.com$)|(^[a-zA-Z0-9._-]*\\.[a-zA-Z0-9._-]*\\.[a-zA-Z0-9._-]*$)|(^[0-9]+(?:\\.[0-9]+){3}(:[0-9]+)?$)"
+        ],
```

Comment:
The pattern needs to be checked. For example, the second group requires entry string dived at least by two dots. I think this is wrong. The third group has no segment length limitation.